content/doc/book/managing/users.adoc

```diff
@@ -1,15 +1,129 @@
 ---
 layout: section
-wip: true
+title: Users
 ---
 ifdef::backend-html5[]
-:notitle:
-:description:
+:description: Learn how to manage users in Jenkins, including creating, configuring, and understanding user management.
 :author:
 :sectanchors:
 :toc:
+:toclevels: 4
 :hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
 endif::[]
 
 = Users
 
+Managing users in Jenkins involves creating, configuring, and maintaining user accounts for individuals who interact with the Jenkins instance. This page provides a step-by-step guide on how to manage users effectively.
+
+== What Managing Users Means
+
+In Jenkins, managing users refers to the process of creating, configuring, and maintaining user accounts. This includes:
+
+- Creating new users.
+- Configuring user details (e.g., full name, email address, password).
+- Modifying user information.
+- Deleting users.
+
+Administrators with the appropriate permissions can perform these actions.
+
+== Accessing the Manage Users Page
```

Comment:
```suggestion
== Accessing the users page
```
Adjusting for sentence casing and the page name would just be "users" since the setting title has changed.